src/openforms/js/components/formio_builder/WebformBuilder.js

```diff
@@ -159,6 +160,7 @@ class WebformBuilder extends WebformBuilderFormio {
             // Context binding
             uniquifyKey={uniquifyKey}
             supportedLanguageCodes={LANGUAGES}
+            theme={currentTheme.value}
```

Comment:
I can't use the `useGlobalState` hook here, but I think this is ok as the light/dark mode can't be toggled when the component editor is opened. Closing the editor, toggling light/dark mode and opening the editor again works as expected.